lib/jekyll/filters.rb

```diff
@@ -357,6 +368,34 @@ def sort_input(input, property, order)
         .map!(&:last)
     end
 
+    private
+    # month_type: "b", "B"
+    # type: nil, "ordinal"
+    # style nil, "US"
+    def stringify_date(date, month_type, type = nil, style = nil)
+      return date if date.to_s.empty?
+      time = time(date)
+      if type == "ordinal"
+        day = time.day
+        ordinal_day = "#{day}#{ordinal(day)}"
+        return time.strftime("%#{month_type} #{ordinal_day}, %Y") if style == "US"
```

Comment:
you forgot to adjust the string interpolation here and below.. :slightly_smiling_face: 
```diff
- time.strftime("%#{month_type}..
+ time.strftime("#{month_type}..
```

Reply:
yes, I was just correcting it :wink: